apps/wallet-dashboard/components/transactions/TransactionTile.tsx

```diff
@@ -100,38 +89,9 @@ function ActivityDetailsDialog({
     open,
     setOpen,
 }: ActivityDetailsDialogProps): React.JSX.Element {
-    const address = useCurrentAccount()?.address ?? '';
-
-    const summary = useTransactionSummary({
-        transaction: transaction.raw,
-        currentAddress: address,
-        recognizedPackagesList: [],
-    });
-
-    if (!summary) return <LoadingIndicator />;
-
     return (
         <Dialog open={open} onOpenChange={setOpen}>
-            <DialogLayout>
-                <Header title="Transaction" onClose={() => setOpen(false)} />
-                <DialogLayoutBody>
-                    <TransactionReceipt
-                        txn={transaction.raw}
-                        activeAddress={address}
-                        summary={summary}
-                        renderExplorerLink={ExplorerLink}
-                        renderValidatorLogo={Validator}
-                    />
-                </DialogLayoutBody>
-                <DialogLayoutFooter>
-                    <ExplorerLink
-                        type={ExplorerLinkType.Transaction}
-                        transactionID={transaction.raw.digest}
-                    >
-                        <ViewTxnOnExplorerButton digest={transaction.raw.digest} />
-                    </ExplorerLink>
-                </DialogLayoutFooter>
-            </DialogLayout>
+            <TransactionDialogDetails transaction={transaction} onClose={() => setOpen(false)} />
```

Comment:
I think that, we can remove component `ActivityDetailsDialog` and move the logic inside `TransactionTile`